modules/plugin/imagepyramid/src/main/java/org/geotools/gce/imagepyramid/ImagePyramidFormat.java

```diff
@@ -206,20 +206,15 @@ public boolean accepts(Object source, Hints hints) {
             }
```

Comment:
I think that other params management (beside overviewPolicy) is out of scope right now (there is no code handling/parsing them). We could consider opening a specific JIRA to add this improvement too.